<?xml version="1.0"?>
<launch>
	<!-- Ros topics -->
	<param name="/planning/cones_topic" value="/trapper/detected"/>
	<param name="/planning/waypoints_topic" value="/planning/waypoints"/>
	<param name="/planning/frame_id" value="rear_link"/>
	<param name="/planning/cpp_cones_topic" value="/planning/cpp_cones"/>
	<param name="/planning/cpp_out_cones_topic" value="/planning/cpp_out_cones"/>
	<param name="/planning/enable_spline" value="false"/>

	<!-- Physical Parameters (/physical/track_width)-->
	<include file="$(find common_params)/launch/common_params.launch"/>

	<!-- planning tree cost parameters -->
	<param name="/planning/track_width_weight" value="1"/>
	<param name="/planning/color_weight" value="1"/>
	<param name="/planning/angle_weight" value="0.1"/>
	<param name="/planning/error_no_detect" value="0.35"/>

	<!-- planning tree search parameters -->
	<param name="/planning/field_of_view" value="3.14"/>
	<param name="/planning/distance" value="6"/>
	<param name="/planning/max_search_iterations" value="50"/>
	<param name="/planning/max_waypoints_per_path" value="5"/>
	<param name="/planning/waypoint_field_of_view" value="3.14"/>
	<param name="/planning/waypoint_distance" value="5"/>
	<param name="/planning/path_queue_limit" value="5"/>
	<param name="/planning/triangulation_radius" value="2.5"/>
	<param name="/planning/triangulation_min_cone_dist" value="1"/>
	<param name="/planning/triangulation_min_waypoint_dist" value="1"/>


	<node name="planner_node" pkg="planning_trees" type="planning_trees_node.py" output="screen"/>
	<node name="cpp_planner_node" pkg="planning_trees" type="ros_node" output="screen"/>
	<include file="$(find waypoint_cleaner)/launch/simple_cleaner.launch"/>
</launch>
